src/services/ExposureNotificationService/ExposureNotificationServiceProvider.tsx

```diff
@@ -55,6 +58,53 @@ export const ExposureNotificationServiceProvider = ({
     });
   }, [backgroundScheduler, exposureNotificationService]);
 
+  const status = exposureNotificationService.exposureStatus.get();
+  const storageService = useStorage();
+  const systemStatus = exposureNotificationService.systemStatus;
+
+  useEffect(() => {
+    const updateExposureStatus = async () => {
```

Comment:
Wondering if we can break this out into a utility function to make it easier to unit test outside of the component.

Reply:
Moving this code from `ExposureNotificationServiceProvider.tsx` to `ExposureNotificationService.ts`.